Library/Formula/mps-youtube.rb

```diff
@@ -0,0 +1,40 @@
+class MpsYoutube < Formula
+  desc "Terminal-based YouTube player and downloader."
+  homepage "https://github.com/np1/mps-youtube"
+  url "https://github.com/mps-youtube/mps-youtube/archive/v0.2.5.tar.gz"
+  sha256 "74d196058c9369a3587f076cafb1ee15baeb6be1465e270f7de9d9830463c869"
+
+  depends_on :python3
+
+  resource "pafy" do
+    url "https://github.com/mps-youtube/pafy/archive/v0.4.2.tar.gz"
+    sha256 "d554d645f9d1e68e4f2b95c586a254fbcb1c4316320cbdec29b48039f496b8d5"
+  end
+
+  resource "youtube-dl" do
+    url "https://github.com/rg3/youtube-dl/archive/2015.11.19.tar.gz"
+    sha256 "ef1925d99d88432afeb02ebde94dec31c977725c4837ce86b7882fbf907f9c57"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python3.5/site-packages"
+
+    res = %w[youtube-dl pafy]
+
+    res.each do |r|
+      resource(r).stage do
+        system "python3", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python3.5/site-packages"
+    system "python3", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "which", "mpsyt"
```

Comment:
@mikemcquaid Alright, so I made it run a download command as a test. However, I'm getting this error:

```bash
Traceback (most recent call last):
  File "/usr/local/Cellar/mps-youtube/0.2.5/libexec/bin/mpsyt", line 9, in <module>
    load_entry_point('mps-youtube==0.2.5', 'console_scripts', 'mpsyt')()
  File "/usr/local/Cellar/mps-youtube/0.2.5/libexec/lib/python3.5/site-packages/mps_youtube/main.py", line 4696, in main
    set_window_title("mpsyt")
  File "/usr/local/Cellar/mps-youtube/0.2.5/libexec/lib/python3.5/site-packages/mps_youtube/main.py", line 152, in set_window_title
    sys.stdout.write(xenc('\x1b]2;' + title + '\x07'))
TypeError: write() argument must be str, not bytes
Error: mps-youtube: failed
Failed executing: mpsyt url https://www.youtube.com/watch?v=qb_hqexKkw8, dv 1 ,\ q
/usr/local/Library/Homebrew/formula.rb:1438:in `block in system'
/usr/local/Library/Homebrew/formula.rb:1375:in `open'
/usr/local/Library/Homebrew/formula.rb:1375:in `system'
/usr/local/Library/Formula/mps-youtube.rb:39:in `block in <class:MpsYoutube>'
/usr/local/Library/Homebrew/formula.rb:1284:in `block in run_test'
/usr/local/Library/Homebrew/extend/fileutils.rb:17:in `mktemp'
/usr/local/Library/Homebrew/formula.rb:1280:in `run_test'
/usr/local/Library/Homebrew/test.rb:28:in `block in <main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/timeout.rb:66:in `timeout'
/usr/local/Library/Homebrew/test.rb:27:in `<main>'
```

I don't have any problems running the same command from within `irb`, so I'm thinking it has to be something with Homebrew. it seems to be the `set_window_title` method...apparently incompatible datatypes.

I have no idea what to do to fix this...